zuul/tenants.yaml

```diff
@@ -53,7 +53,6 @@
           - ansible-community/pytest-molecule
           - ansible/network
           - ansible-network/arista_eos
-          - ansible-network/aws
```

Comment:
lets move this into another PR,  if no longer needed. As we need to retire via github/projects.yaml